pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -53,9 +53,16 @@ const (
 	Success Code = iota
 	// Error is used for internal plugin errors, unexpected input, etc.
 	Error
-	// Unschedulable is used when a plugin finds a pod unschedulable.
+	// Unschedulable is used when a plugin finds a pod unschedulable. The scheduler might attempt to
+	// preempt other pods to get this pod scheduled. Use UnschedulableAndUnresolvable to make the
+	// scheduler skip preemption.
 	// The accompanying status message should explain why the pod is unschedulable.
 	Unschedulable
+	// UnschedulableAndUnresolvable is used when a (pre-)filter plugin finds a pod unschedulable and
```

Comment:
Unless this PR will be merged into 1.16, it would be better to append `UnschedulableAndUnresolvable` to the last position since we use `iota` here. But it is ok if we decide to break it as we are in the alpha stage.